File: my-node.js
class MyNode {
    constructor(value,
        link=null) {
            this.value = value;
            this.link = link;
        }
}


var a = new MyNode(0);
console.log(a);
console.log(a.value);
console.log(a.link);
a.link = new MyNode(1);
console.log(a.link);

File: my_node.py
class MyNode:
    def __init__(self, value, link=None):
        self.value = value
        self.link = link

    def __str__(self):
        return f"node with val {self.value}"

a = MyNode(0)
a.link = MyNode(1)
print(a)
print(a.link)
